homeassistant/components/alarm_control_panel/__init__.py

```diff
@@ -164,6 +164,11 @@ def async_alarm_arm_custom_bypass(self, code=None):
         """
         return self.hass.async_add_executor_job(self.alarm_arm_custom_bypass, code)
 
+    @property
+    def supported_features(self) -> int:
```

Comment:
Better is to mark it as `@abstractmethod` imported from `abc` package

Reply:
Also update the calls to `async_register_entity_service` to pass in the feature that is required for that service.

Reply:
Do you have any examples for async_register_entity_service .....?

Reply:
It's already used above, you just need to pass in the supported feature that is required for the service to be allowed to call.

Reply:
Thanks, done.